OpenRA.Mods.Common/Widgets/Logic/Editor/ActorEditLogic.cs

```diff
@@ -491,7 +491,7 @@ public void Execute()
 
 		public void Do()
 		{
-			actor = editorActorLayer[actorId.ToLowerInvariant()];
+			actor = editorActorLayer[actorId];
```

Comment:
won't this cause errors?

Reply:
The lookup is already case insensitive, so calling `ToLowerInvariant` is unnecessary.

```c#
public EditorActorPreview this[string id]
{
	get { return previews.FirstOrDefault(p => p.ID.Equals(id, StringComparison.OrdinalIgnoreCase)); }
}
```